api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -625,6 +630,8 @@ private boolean userNeverLoggedIn(
     validateProfileFields(updatedProfile);
     DbUser user = userProvider.get();
 
+    // save previous profile for audit trail
```

Comment:
I don't think we need this comment, we can always follow usages and it has a high chance of getting out of date if this variable is used in other ways